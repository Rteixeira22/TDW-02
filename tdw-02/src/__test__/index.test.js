import React from 'react';
// Importing the render method from the @testing-library/react library
import { render } from '@testing-library/react';
// Importing the Home component from the pages directory
import Home from '../pages/index';

// Defining a test case named 'renderiza corretamente'
test('renderiza corretamente', () => {
  // Rendering the Home component and destructuring the asFragment method from the result
  const { asFragment } = render(<Home />);
  // Asserting that the rendered output matches the snapshot
  expect(asFragment()).toMatchSnapshot();
});