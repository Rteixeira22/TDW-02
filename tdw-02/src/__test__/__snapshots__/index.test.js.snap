// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderiza corretamente 1`] = `
<DocumentFragment>
  <div>
    <main>
      <h1>
        Test c/ jest
      </h1>
      <h3>
        Test pipeline
      </h3>
      <li>
         Teste de prettier
      </li>
       
      <li>
        mais um teste 
      </li>
       
      <li>
         Teste de prettier
      </li>
       
      <li>
        maais um teste 
      </li>
       
      <li>
         Teste de prettier
      </li>
       
      <li>
        maais um teste 
      </li>
       
      <li>
         Teste de prettier
      </li>
       
      <li>
        maais um teste 
      </li>
       
      <li>
         Teste de prettier
      </li>
       
      <li>
        maais um teste 
      </li>
       
      <li>
         Teste de prettier
      </li>
       
      <li>
        maais um teste 
      </li>
    </main>
    <footer />
  </div>
</DocumentFragment>
`;
